RFS/README.md

```diff
@@ -74,18 +74,18 @@ gradle run --args='-n global_state_snapshot --source-host $SOURCE_HOST --source-
 ### Using Docker
 RFS has support for packaging its java application as a Docker image by using the Dockerfile located in the `RFS/docker` directory. This support is directly built into Gradle as well so that a user can perform the below action, and generate a fresh Docker image (`migrations/reindex_from_snapshot:latest`) with the latest local code changes available.
 ```shell
-./gradlew buildDockerImages
+../gradlew buildDockerImages
```

Comment:
What about indicating moving to the project root, or not including the path directive at all?
```suggestion
cd {PROJECT_ROOT}
./gradlew buildDockerImages
```

Reply:
what if I added this to each of the three READMEs or the top README?  It's long, but it works all through the tree.

`agradle() { local dir=$(pwd); while [[ "$dir" != "/" ]]; do [[ -x "$dir/gradlew" ]] && "$dir/gradlew" "$@" && return; dir=$(dirname "$dir"); done; echo "No gradlew script found in the ancestor directories"; }`

Reply:
This seems like it creates more complexity than ease of use.  IMO its much simpler to force everything to be done from the root of the project, at least when you can get away with a single purpose oriented repo

Reply:
I agree that the agradle thing is scary and off-putting.  I appreciate that '..' is unambiguous and won't affect the user's current environment as `cd` would.

Reply:
Resolving this thread, I'm happy to revisit this post merge myself if I've got other ideas